apps/emqx_management/src/emqx_mgmt.erl

```diff
@@ -919,7 +919,7 @@ get_telemetry_data() ->
 
 item(client, {ClientId, ChanPid}) ->
     Attrs = case emqx_cm:get_chan_info(ClientId, ChanPid) of
-                undefined -> #{};
+                undefined -> throw(gone);
```

Comment:
what's the result of this change ?
does it make sense to be more informative about the exception ?
e.g. `throw({non_existing_channel, ClientId})` ?